pkg/ccl/importccl/read_import_base.go

```diff
@@ -62,7 +62,7 @@ func runImport(
 			inputs = make(map[int32]string)
 			for id, name := range spec.Uri {
 				// TODO(yevgeniy): Support offsets into the file, not just full file skipping.
-				if seek, ok := spec.ResumePos[id]; !ok || seek != math.MaxUint64 {
+				if seek, ok := spec.ResumePos[id]; !ok || seek != math.MaxInt64 {
```

Comment:
I wonder if we should also check against `int64(math.MaxUint64)` for upgrade compatibility? I guess worst-case is we re-do more work than needed on upgrade?

Reply:
Up until now, the only time resume position was set to non-zero value is when the import finished fully (maxuint64).

If the resume was paused, the resume position was always set 0.
Since we can only resume paused jobs, then I think we can only see a 0 here.
As matter of fact, I kinda think that this check and filtering is an overkill right now since we only mark files fully processed when we finish entire import (and not after each file).
Nonetheless, we could decide to do so in the future.

I think we're okay w/out uint64 check

Reply:
We might not have paused -- we might have just had the coordinator vanish. In either case, some of the files could have finished before we paused/vanished and have written `math.MaxUnit64` to their persisted progress, right?

I'm just thinking that when we load progress from the job, we might need to check for that (either here or where we read the job record)?

Reply:
You're right -- forgot about multiple node imports... We verified that a switch from uint -> int64 would result in resume position being represented as a huge negative number, which will have the effect of redoing the work (during the upgrade), which is fine